lib/npm/routes.js

```diff
@@ -59,6 +53,14 @@ module.exports = function (app) {
    * definitions:
    *   DistTags:
    *     type: object
+   * parameters:
```

Comment:
Why can't be this with the others in `lib/routes/swagger.js`?

Reply:
The general strategy I was trying to follow here was:

If it's used by a single route, just document it there,
If it's used by a single file, put the shared definition in that file,
If it's used by more than one file, put the shared definition in `lib/routes/swagger.js`

Could change that and move all the shared definitions into swagger.js I'm not sure I feel too strongly about it

Reply:
I would prefer something like:
If it's used by a single route, just document it there,
If it's used by more than once then in the shared location.

I was just curious, not a blocking / strong point.